model/riscv_vmem_sv32.sail

```diff
@@ -24,7 +24,7 @@ function walk32(vaddr, ac, priv, mxr, do_sum, ptb, level, global, ext_ptw) = {
     MemValue(v) => {
       let pte = Mk_SV32_PTE(v);
       let pbits = pte.BITS();
-      let ext_pte : extPte = zeros(); // no reserved bits for extensions
+      let ext_pte : extPte = init_sv32_ext_pte;
```

Comment:
Yes, this would now allow overriding by an extension.   Should we use `default_sv32_ext_pte` though?  I'd also put back the deleted comment but explain that there is a default for extension bits since there are no reserved bits.